### PR TITLE
Release-1.4 : post-release files updated

### PR DESCRIPTION
**Pull request by mskanth972**
**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**

**What testing is done?**

### CODE REVIEW

File: charts/aws-efs-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-efs-csi-driver
 version: 2.2.8
```
Comment (Ashley-wenyizha) [inline charts/aws-efs-csi-driver/Chart.yaml:3]: Please bump up helm chart version here

Comment (mskanth972) [reply charts/aws-efs-csi-driver/Chart.yaml:3]: Sure

**Comment by Ashley-wenyizha** [comment]
Please squash the commits and only keep one commit

**Comment by Ashley-wenyizha** [comment]
approved

**Pull request merged**